dashboard/app/models/script.rb

```diff
@@ -158,7 +159,7 @@ def starting_level
 
   # Caching is disabled when editing scripts and levels or running unit tests.
   def self.should_cache?
-    return false if Rails.application.config.levelbuilder_mode
+    return false unless Rails.application.config.enable_script_cache
```

Comment:
I'm not sure I love the idea of caching defaulting to false; I'd much rather have things default to how we want them in production.